Elevel/Application/Features/AuditionCommands/UpdateAuditionCommand.cs

```diff
@@ -111,6 +113,8 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
                     throw new NotFoundException(nameof(Audition), request.Id);
                 }
 
+                audition.Deleted = true;
```

Comment:
I don't like this. You are not deleting it, it's archive, + I think it will be good to have some correlationId to combine all question states across versions.